### ISSUE TITLE
New variable attribute: similar variable

### ISSUE THREAD

**Issue by anfranken** [dzhw/metadatamanagement#724]
- for generated variables
- for variables of other surveys 
- for variables of the same survey which are no panel variables
- ???

**Comment by rreitmann** [comment]
@anfranken Isn't this variable.relatedVariables? If yes, please close this issue...

**Comment by anfranken** [comment]
Yes, but the idea of that issue was to partition variable.relatedVariables in many subattributes. At the moment it is enough if we have on attribute for all, but in future it will be nice if we could distinguish between the different kinds of related variables.

**Comment by UteH** [comment]
related to #1229
